.github/workflows/deploy.yml

```diff
@@ -34,10 +34,6 @@ jobs:
         run: sudo npm i yalc -g
       - name: NPM Install
         run: npm ci
-      - name: Deploy the Serverless Demo App
```

Comment:
Hmm this is not what we want. We need to git clone the meeting demo and deploy it :)

Reply:
Do you mean clone the single folder? Please correct me if I am wrong:
```
name: Deploy the Serverless Demo App
run: svn checkout https://github.com/aws-samples/amazon-chime-sdk/trunk/apps/meeting
    cd $GITHUB_WORKSPACE/apps/meeting/serverless
    node ./deploy.js -r us-east-1 -b chime-sdk-components-demo-canary -s chime-sdk-components-demo-canary -l

Reply:
Use this action instead https://github.com/aws/amazon-chime-sdk-js/blob/master/.github/workflows/content-share-integration.yml#L22 and point it to the new meeting demo location.

Reply:
example: https://github.com/aws/amazon-chime-sdk-js/actions/runs/944457360/workflow#L17-L20

Reply:
Thanks for your advice, updated